lib/modules/dosomething/dosomething_signup/includes/SignupTransformer.php

```diff
@@ -78,19 +83,13 @@ protected function transform($item) {
    * @param  array  $parameters
    * @return array
    */
-  private function setFilters($parameters) {
-    $filters = [
-      'sid' => dosomething_helpers_format_data($parameters['ids']),
-      // 'users' => dosomething_helpers_format_data($parameters['users']),
-      // 'nid' => dosomething_helpers_format_data($parameters['campaigns']),
-    ];
+  // private function setFilters($parameters) {
```

Comment:
I'd recommend using the `setFilters()` function for the params... cause the list of params could grow over time and nice to split that out into its own function :smiley: